core/src/main/kotlin/backend/team/ahachul_backend/api/lost/domain/entity/LostPostEntity.kt

```diff
@@ -108,6 +109,10 @@ class LostPostEntity(
         category?.let { this.category = category }
     }
 
+    fun updateStatus(command: UpdateLostPostStatusCommand) {
```

Comment:
엄청 사소하기는 한데, 어짜피 내부에서 사용하는 인자가 하나라 command 말고 status 값으로 받아도 좋을 것 같아요!
```suggestion
    fun updateStatus(status: LostStatus) {
```

Reply:
@semi-cloud 네! command 안넘겨도 될 것 같아서 수정하였습니다 👍